src/soap/tests/test_client.py

```diff
@@ -159,3 +237,33 @@ def test_default_behaviour_relative_paths(self, m):
             session.post("api/relative")
 
         self.assertEqual(m.last_request.url, "https://example.com/api/relative")
+
+    def test_it_can_build_a_functional_client(self):
+        service = SoapServiceFactory.build(
+            url="http://www.soapclient.com/xml/soapresponder.wsdl"
+        )
+        client = build_client(service)
+
+        self.assertEqual(
+            client.service.Method1("under the normal run", "things"),
+            "Your input parameters are under the normal run and things",
+        )
+
+
+class ClientTransportTimeoutTests(TestCase):
+    @override_settings(DEFAULT_TIMEOUT_REQUESTS=1)
+    def test_the_client_obeys_timeout_requests(self):
```

Comment:
check `openforms.tests.test_requests_timeout` for the `@skipIf` so that I can still run these tests offline in the train :wink: 

edit: actually it'll pass I think because the `ConnectionError` is a `RequestException` :thinking:

Reply:
It is.